includes/derivatives.inc

```diff
@@ -305,14 +305,14 @@ function islandora_ocr_derive_ocr(AbstractObject $object, $force = FALSE) {
  */
 function islandora_ocr_derive_hocr(AbstractObject $object, $force = FALSE) {
```

Comment:
Could dedup this to a single parameterized function... Could then avoid issues such as down on line 353 (referring to "OCR").